.github/workflows/deploy-helm-charts.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release Helm Charts
+
+on:
+  release:
+    # Only cut new helm charts when a new release is published
+    types: [published]
```

Comment:
Maybe for this PR, we can change this [to include `workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch). That way, the maintainers with access rights can test this workflow with `gh` on the command line without having to cut arbitrary releases:

```
gh workflow run deploy-helm-chart.yml
```

Reply:
I am okay with including this in a separate PR, we just want it before we cut this into a release.